src/app/dashboard/components/chart/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  LineChart,
+  Line,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts';
+import styled, { useTheme } from 'styled-components';
+
+import { ThemeType } from 'theme/defaults';
+
+import { StockChartData } from '../../sampleChartData';
+
+function linspace(start: number, stop: number, num: number, endpoint = false) {
+  const div = endpoint ? num - 1 : num;
+  const step = roundTo10((stop - start) / div);
+  return Array.from({ length: num }, (_, i) => start + step * i);
+}
+
+function roundTo10(value: number) {
+  return Math.round(value / 10) * 10;
+}
+
+type Props = {
+  data: StockChartData[];
+};
+
+const Container = styled.div`
+  height: 32rem;
+  padding: 2rem;
+  margin: 2rem;
+`;
+
+const StockChart: React.FC<Props> = ({ data }) => {
+  const theme = useTheme() as ThemeType;
+  const isAggregate: boolean = data[0].low !== undefined;
```

Comment:
TS sam domyśli się typu - nie trzeba `:boolean`

```suggestion
  const isAggregate = data[0].low !== undefined && data[0].high !== undefined;
```